joi-validations/topics.validations.js

```diff
@@ -27,6 +27,12 @@ const TopicValidation = {
       member_ids: array(string.required()),
       topic_id: string.required()
     }
+  },
+  getTopic: {
+    query: {
+      name: string.min(3),
+      withMinimumFollower: boolean
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `getTopic` validation schema is a good practice to ensure that the incoming query parameters are valid. However, the `name` field should be required as well, since it's essential for retrieving a topic. Also, the `withMinimumFollower` field should be an integer (representing the minimum follower count), not a boolean.
```diff
- getTopic: {
-   query: {
-     name: string.min(3),
-     withMinimumFollower: boolean
-   }
- }
+ getTopic: {
+   query: {
+     name: string.min(3).required(),
+     withMinimumFollower: number.min(0)
+   }
+ }
```
Please replace `number` with the appropriate method for validating numbers in your validation library.


<!-- This is an auto-generated comment by OSS CodeRabbit -->